recipes/install.rb

```diff
@@ -49,11 +49,7 @@
 
 powershell_script 'Unzip Winlogbeat' do
   code <<-EOH
-  $shell = new-object -com shell.application
-  $zip = $shell.NameSpace('#{package_file}')
-  foreach($item in $zip.items()) {
-    $shell.Namespace('#{destination}').copyhere($item)
-  }
+    Expand-Archive '#{package_file}' '#{destination}'
```

Comment:
This cmdlet is available from Powershell 5.1 only. Older version  won't be supported. What do you think?